<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-ICG6BLL" failures="2" tests="2" name="Default test(failed)" time="0.09" errors="0" timestamp="2020-11-29T22:22:22 IST">
  <testcase classname="com.testng.learning.ListeningClass" name="t1" time="0.017">
    <failure type="java.lang.AssertionError" message="Test case Passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test case Passed expected [true] but found [false]
at com.testng.learning.ListeningClass.t1(ListeningClass.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- t1 -->
  <testcase classname="com.testng.learning.ListeningClass" name="t2" time="0.003">
    <failure type="java.lang.AssertionError" message="Test case Passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Test case Passed expected [true] but found [false]
at com.testng.learning.ListeningClass.t2(ListeningClass.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- t2 -->
</testsuite> <!-- Default test(failed) -->
